### PR TITLE
feat(web): fix feedback PR (clean code)

### PR DESCRIPTION
**Pull request by darsene**
ref: MANAGER-12167

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/dkim-b2`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-12167
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- ~Only FR translations have been updated~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~Breaking change is mentioned in relevant commits~

### CODE REVIEW

**Review by antonymarion — APPROVED** [review]

File: packages/manager/modules/emailpro/src/domain/dkim-autoconfig/dkim-auto-configurator.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ export default class DkimAutoConfigurator {
     this.showSpfDiagnosticTitle = false;
     this.loading = true;
     this.dkimSelectorsNoDomain = await this.getDkimSelectorForCurrentState();
-    await this.stepConfigureDkimFor('emailpro');
+    await this.stepConfigureDkim();
```
Comment (anooparveti) [inline packages/manager/modules/emailpro/src/domain/dkim-autoconfig/dkim-auto-configurator.js:46]:
```suggestion
    await this.confidureDKIMStep();
```

File: packages/manager/modules/emailpro/src/domain/dkim-autoconfig/dkim-auto-configurator.js [hunk 2]
```diff
@@ -123,17 +123,17 @@ export default class DkimAutoConfigurator {
...
   }
 
-  writeMessage(msg, type, args) {
-    const message = this.services.$translate.instant(msg, args);
-    this.services.$scope.setMessage(message, {
+  writeMessage(message, type, params) {
+    const translation = this.services.$translate.instant(message, params);
+    this.services.$scope.setMessage(translation, {
       type,
     });
```
Comment (anooparveti) [inline packages/manager/modules/emailpro/src/domain/dkim-autoconfig/dkim-auto-configurator.js:138]: `writeError` -> `setErrorMesssage`, `writeSuccess` -> `SetSuccessMessage`
I dont see the need for `writeMessage` wrapper.

File: packages/manager/modules/exchange/src/domain/dkim-autoconfig/dkim-auto-configurator.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ export default class DkimAutoConfigurator {
     this.showSpfDiagnosticTitle = false;
     this.loading = true;
     this.dkimSelectorsNoDomain = await this.getDkimSelectorForCurrentState();
-    await this.stepConfigureDkimFor('exchange');
+    await this.stepConfigureDkim();
```
Comment (anooparveti) [inline packages/manager/modules/exchange/src/domain/dkim-autoconfig/dkim-auto-configurator.js:47]:
```suggestion
    await this.configureDKIMStep();
```

File: packages/manager/modules/emailpro/src/domain/dkim-autoconfig/dkim-auto-configurator.js (hunk 2 above)
Comment (darsene) [reply packages/manager/modules/emailpro/src/domain/dkim-autoconfig/dkim-auto-configurator.js:138]: DKIM has a lot of duplicated code, so to ease improvements or a potential refactoring, the code is kept quite similar to exchange part (https://github.com/ovh/manager/blob/feat/MANAGER-12167/packages/manager/modules/exchange/src/domain/dkim-autoconfig/dkim-auto-configurator.js#L68)

File: packages/manager/modules/exchange/src/domain/dkim-autoconfig/dkim-auto-configurator.js (hunk 3 above)
Comment (antonymarion) [reply packages/manager/modules/exchange/src/domain/dkim-autoconfig/dkim-auto-configurator.js]: configureDKIMStep has been renamed configureDKIM

File: packages/manager/modules/emailpro/src/domain/dkim-autoconfig/dkim-auto-configurator.js (hunk 1 above)
Comment (antonymarion) [reply packages/manager/modules/emailpro/src/domain/dkim-autoconfig/dkim-auto-configurator.js]: configureDKIMStep has been renamed configureDKIM

**Review by anooparveti — APPROVED** [review]

**Pull request merged**